### PR TITLE
Add description of draggable visualisations to changelog.

### PR DESCRIPTION
**Pull request by MichaelMauderer**
### Pull Request Description
Changelog update for previous PR.

### Checklist
Please include the following checklist in your PR:

- [x] The `CHANGELOG.md` was updated with the changes introduced in this PR.
- [ ] ~The documentation has been updated if necessary.~
- [ ] ~All code conforms to the [Rust](https://github.com/enso-org/enso/blob/main/docs/style-guide/rust.md) style guide.~
- [ ] ~All code has automatic tests where possible.~
- [ ] ~All code has been profiled where possible.~
- [ ] ~All code has been manually tested in the IDE.~
- [ ] ~All code has been manually tested in the "debug/interface" scene.~
- [ ] ~All code has been manually tested by the PR owner against our [test scenarios](https://docs.google.com/spreadsheets/d/1RatJDM_f9_3bvYhl3Bpq2d8SyKgtVdrV1RkGxPU17c8/edit?ts=5faa7049#gid=0).~
- [ ] ~All code has been manually tested by at least one reviewer against our [test scenarios](https://docs.google.com/spreadsheets/d/1RatJDM_f9_3bvYhl3Bpq2d8SyKgtVdrV1RkGxPU17c8/edit?ts=5faa7049#gid=0).~

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ collection of all data, respectively. To learn more what data is collected in En
 read the notes of the `Enso 2.0.0-alpha.1` release.
 
 
+<br/>![New Features](/docs/assets/tags/new_features.svg)
+
+#### Visual Environment
+- [Added the ability to reposition visualisations.][1096] There is no an icon in the visualisation action
+  bar that allows dragging the visualisation. Once it has been moved there appears another icon that
+  will reset the position to the original position.
```
Comment (wdanilo) [inline CHANGELOG.md]: 1. There is a typo "no" -> "now"
2. There are 3 other English errors catched by Gramarly. Please, create a free Gramarly account and check your spelling there before changing CHANGELOG. Btw, I started recently doing it for my code comments as well - this automatic tool is amazing (I was using it for years when writing mails already)

**Review by wdanilo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (MichaelMauderer) [reply CHANGELOG.md]: `LanguageTool` is better (multi-language support) and seems to even have an [IDE integration](https://plugins.jetbrains.com/plugin/12175-grazie)! So, I'm using that now. Much better than to having to paste back and forth.

**Review by wdanilo — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,17 @@ collection of all data, respectively. To learn more what data is collected in En
...
 
 
+<br/>![New Features](/docs/assets/tags/new_features.svg)
+
+#### Visual Environment
+- [Added the ability to reposition visualisations.][1096] There is now an icon in the visualization 
+  action bar that allows dragging the visualization. Once the visualization has been moved, there 
+  appears another icon that will reset the position to the original position.
+
+[1096]: https://github.com/enso-org/ide/issues/1096
```
Comment (wdanilo) [inline CHANGELOG.md]: one more thing - it'd be good if the link will lead to Pr with animations instead of just task description

**Review by wdanilo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (MichaelMauderer) [reply CHANGELOG.md]: Updated.

**Pull request merged**